src/support/mixins/misc.scss

```diff
@@ -24,3 +24,33 @@
     background-color: $border;
   }
 }
+
+// global focus styles
+
+@mixin focusOutline {
+  z-index: 1;
+  outline: 2px solid var(--color-accent-fg);
+  outline-offset: 2px;
+}
+
+@mixin minTouchTarget(
```

Comment:
@vdepizzol do you have an opinion on how granular this mixin should be? I see two options:

One that handles `pointer: coarse` abstractly in the mixin, so consumer passes in two sets of values (or uses defaults)

```
@mixin minTouchTarget(
  $min-height-desktop: 32px,
  $min-width-desktop: 32px,
  $min-height-mobile: 44px,
  $min-width-mobile: 44px
) {
  content: '';
  position: absolute;
  width: 100%;
  height: 100%;
  min-height: $min-height-mobile;
  min-width: $min-width-mobile;
  top: 50%;
  left: 50%;
  transform: translateX(-50%) translateY(-50%);

  @media (pointer: coarse) {
    min-height: $min-height-desktop;
    min-width: $min-width-desktop;
  }
}
```

Or let the component CSS decide to add desktop styles

```
@mixin minTouchTarget(
  $min-height: 44px,
  $min-width: 44px
) {
  content: '';
  position: absolute;
  width: 100%;
  height: 100%;
  min-height: $min-height;
  min-width: $min-width;
  top: 50%;
  left: 50%;
  transform: translateX(-50%) translateY(-50%);
}

// my css

.someClass {

  @media (pointer: coarse) {
    @include minTouchTarget(32px, 32px)
  }
}
```